plugins/vuetify.js

```diff
@@ -1,13 +1,48 @@
 import "vuetify/styles";
 import { createVuetify } from "vuetify";
-import { aliases, mdi } from "vuetify/iconsets/midi";
+import '@mdi/font/css/materialdesignicons.css';
```

Comment:
Looks like we didn't get to use any icon from this import. I'd suggest we take it off for now.